src/ima.js

```diff
@@ -454,6 +455,20 @@ class Ima extends BasePlugin implements IMiddlewareProvider, IAdsControllerProvi
     });
   }
 
+  /**
+   * set default Configuration for plugin
+   * @private
+   * @returns {void}
+   * @instance
+   * @memberof Ima
+   * @param {Object} config - configuration of app for ima plugin
+   */
+  _setDefaultConfiguration(config: Object): void {
```

Comment:
how does this relate to the issue of ads covering entire page?

Reply:
It's part of FEC-9013, It moved to(https://github.com/kaltura/playkit-js-ima/pull/103)